h2o-dist/index.html

```diff
@@ -247,6 +247,35 @@
       .pre {
           white-space: pre;
       }
+      
+      .callout {
```

Comment:
I added callout styles. Here is what they look like:

<img width="1029" alt="Screenshot 2023-08-10 at 2 08 27 PM" src="https://github.com/h2oai/h2o-3/assets/641147/96aa2fb0-a4e4-42c5-ac87-22c8e4035de2">

Reply:
@jefffohl Thank you -- this looks so nice! Is it cool if I update the color so it's less panicky? Perhaps, these blues? 
<img width="990" alt="Screen Shot 2023-08-10 at 4 30 27 PM" src="https://github.com/h2oai/h2o-3/assets/52463461/7a475e91-a09f-40be-8e5f-91083e0aaeda">

Reply:
Yes, sure - no problem - looks nice.